### DISCUSSION TITLE
Option to Enable All Filters

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#1749] [Ideas]
Can you please add an option to enable all filters button? When using multiple browsers and profiles, it is very annoying having to check every single checkbox by hand. Yes this is the only way I run uBlock, by using all filters. I'm sure many do the same. Thank you!

**Comment by uBlock-user** [comment]
Already declined once by @gorhill -- https://github.com/uBlockOrigin/uBlock-issues/issues/172#issuecomment-413836348

**Reply by ghost** [reply]
@uBlock-user Oh, you know and I actually searched if I posted it before *forgot* =( Meh! Said!
